app/redux/reducers/ConnectedDeviceReducer.js

```diff
@@ -69,8 +74,14 @@ const ConnectedDeviceReducer = ( state = defaultState, action) => {
             });
         case RECONNECTING_TO_DEVICE:
             return Object.assign({}, state, {
-                isReconnecting: true
+                isReconnecting: true,
+                numReconnects: state.numReconnects += 1
```

Comment:
Might not want to listen to RECONNECTING_TO_DEVICE, instead listen to RECONNECT_DEVICE as that runs when it actually reconnects rather than when it enters reconnecting mode

Furthermore, should be +1 not +=1

Reply:
done